convert.py

```diff
@@ -4,11 +4,16 @@
 import sys
 import os
 import tempfile
+import logging
+
+# Change the level from "logging.INFO" to "logging.DEBUG" on "basicConfig" line below to get more info
+# As an alternative to calling this script with "2>1", you can add ",filename=<filename.log>" after the level is specified in the "basicConfig" method below 
+logging.basicConfig(level=logging.INFO, format='%(asctime)s - %(levelname)s - %(message)s')
```

Comment:
Cool, good idea! You could read some environment variable to change the log level so that one does not need to change this file.

Reply:
Love it. For some of the next changes I was planning on implementing a config file anyway. I was originally thinking of putting it in GSWL-private so that people can change things in their private whilst maintaining a clean copy of the ecosystem... Is that a good idea? Or would you prefer the 'convert.py' config within the ecosystem repo?